app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -77,6 +80,13 @@ public void onUsernameAlreadyStored(@NonNull String usernameStored) {
         }
     }
 
+    @Override
+    public void onPasswordAlreadyStored(@NonNull String passwordStored) {
+        if (password != null) {
```

Comment:
If `passwordStored` is NonNull why are you evaluatin if it is null? :thinking:

Reply:
that case is for the password edittext, verify if is null because didn't attach the view at first

Reply:
I'm sorry I've confused `password` with `passwordStored`